wasp/boards/p8/manifest.py

```diff
@@ -11,7 +11,7 @@
     (
         'boot.py',
         'draw565.py',
-        'drivers/bma421.py',
+        'drivers/sc7a20.py',
```

Comment:
Just a quick comment but there are plenty of P8 boards with BMA421 sensors (in fact I wear one most of the time). You probably need to create a second board type (P8b) for the newer devices.

Reply:
You are totally right 🤦